ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/disconnected/provision/cucushift-installer-rehearse-vsphere-ipi-disconnected-provision-chain.yaml

```diff
@@ -18,5 +18,8 @@ chain:
   - name: BASTION_HOST_SUBNET
     default: "qe-segment"
     documentation: the subnet to provision bastion host
+  - name: "STEPUP_IMAGE_REGISTRY_WITH_PVC"
```

Comment:
Suggest to create a dedicated folder to save the workflow, create it under `vsphere/ipi/disconnected/image-registry-pvc/`

Reply:
Just like what we did for fips: https://github.com/openshift/release/tree/master/ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/proxy/fips

Reply:
@jianlinliu  if it's ok that just add the workflow in 4.12 and 4.13 amd64 nightly files,  and set p2 and f7 ?

Reply:
My suggestion is adding a new install workflow for this test scenario, while in our daily ci yam files, we replace all the occurrence of  `vsphere-ipi-disconnected` with this new workflow.